File: example/twitter_flux/lib/views/login_view.dart
import 'package:flutter/material.dart';

// TODO: 由你来实现登录页面 UI
// 调用 fluxClient.emit("auth/login", {"username": ..., "password": ...})

class LoginView extends StatelessWidget {
  final VoidCallback onLoginSuccess;

  const LoginView({super.key, required this.onLoginSuccess});

  @override
  Widget build(BuildContext context) {
    return const Scaffold(
      body: Center(child: Text('Login View - TODO')),
    );
  }
}

File: example/ios/TwitterFlux/TwitterFlux/Views/LoginView.swift
import SwiftUI
import FluxSDK

// TODO: 由你来实现登录页面 UI
// 调用 store.emit("auth/login", payload: LoginPayload(username: ..., password: ...))
// 登录成功后 store.authState.isLoggedIn 会变为 true，自动跳转到主页面

struct LoginView: View {
    @EnvironmentObject var store: FluxStore

    var body: some View {
        Text("Login View - TODO")
    }
}
